TwoPaneLayout/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
```

Comment:
I think you can use a better `.gitignore` rule set like this: https://github.com/microsoft/surface-duo-compose-samples/blob/main/.gitignore